redash/query_runner/oracle.py

```diff
@@ -141,10 +152,7 @@ def run_query(self, query, user):
                         for i in cursor.description
                     ]
                 )
-                rows = [
-                    dict(zip((column["name"] for column in columns), row))
-                    for row in cursor
-                ]
+                rows = [dict(zip((c["name"] for c in columns), row)) for row in cursor]
```

Comment:
This feels harder to read than necessary.

```suggestion
                columns = [i.get("name") for i in columns]
                rows = [
                    {field: value for field, val in zip(columns, row)} for row in cursor
                ]
```

Reply:
Black formatter is to blame on that 😬